File: api/Controllers/Models/Entities/Invoice.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations.Schema;

namespace IPKP___API.Controllers.Models.Entities
{
    public class Invoice
    {
        [Key]
        public Guid Invoice_ID { get; set; }

        [ForeignKey(nameof(Customer_ID))]
        public Guid Customer_ID { get; set; }

        public double Discount_Amount { get; set; }
        public double Delivery_Price { get; set; }
        public double Invoice_Total_exclVAT { get; set; }
        public double Invoice_Total_VAT { get; set; }
        public double Invoice_Total_inclVAT { get; set; }
    }
}

File: customer/src/app/Models/invoice.ts
import { Order_Request } from "./orderrequest";
import { Invoice_Discount } from "./invoicediscount";

export class Invoice{
    invoice_ID!: string   
    delivery_Price!: Number
    invoice_Total_exclVAT!: Number
    invoice_Total_VAT!: Number
    invoice_Total_inclVAT!: Number

    order_Request!: Order_Request
    invoice_Discount!: Invoice_Discount
}